app/src/main/java/com/ikarabulut/Board.java

```diff
@@ -49,4 +51,142 @@ public void setRandomGrid() {
     public List<ArrayList> getGrid() {
         return grid;
     }
+
+    public void setAliveAt(int rowNum, int colNum) {
```

Comment:
One thing that is nice about statically typed languages that is different from dynamic ones is that variables are annotated with their type. In this case `rowNum` and `colNum` are both of type `int`.  That means we can infer that they are, in fact, numbers.  That makes the `Num` suffix for these variable names redundant.

This is a common thing people do at first, because they are used to dynamic languages where they may not know the type of a variable by looking at it.  Adding an object's type to it's name is called [Hungarian Notation](https://en.wikipedia.org/wiki/Hungarian_notation) and is, in general, discouraged because it makes variables harder to read.